packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -385,3 +385,24 @@ export interface IGCResult {
 	/** The ids of nodes that are not-referenced or deleted in the referenced graph */
 	deletedNodeIds: string[];
 }
+
+/*
+ * Data type for ContainerRuntime "lifecycle" op contents
+ */
+export type ObjectLifecycleMessage =
```

Comment:
Existing "attach" op fits in here too.  Makes me think of potentially migrating that later, or on the other hand just ditching the "lifecycle" layer and adding "reference" and "delete" to ContainerMessageType directly